src/main/java/frc/robot/RobotContainer.java

```diff
@@ -165,7 +169,7 @@ private void configureButtonBindings() {
     m_coDriverController.a().whileTrue(prepClose);
     m_coDriverController.x().whileTrue(prepAmp);
     m_coDriverController.b().whileTrue(new Puke()).onFalse(new ResetWheels());
-    m_coDriverController.leftTrigger().onTrue(new SimpleScoreTest());
+    m_coDriverController.leftTrigger().onTrue(new SimpleScoreAdjust());
```

Comment:
Do we want to keep this here? Or are you done testing and want to revert this

Reply:
buttons are not decided yet we need additional testing tbh but we are keeping interpolate and the normal scoring buttons we already have